### ISSUE TITLE
Old documentation site still around

### ISSUE THREAD

**Issue by astrojuanlu** [executablebooks/thebe#518]
I accidentally (perhaps because I had visited it earlier this year?) arrived to `https://thebelab.readthedocs.io/` and I was about to report here that there was no `#activateButton`, but then I realized that the up to date documentation of the project lives at https://thebe.readthedocs.io/.

I wonder if you would like to create a redirection from one project to another, to avoid confusion?

**Comment by stevejpurves** [comment]
@astrojuanlu thanks for reporting that! A redirect makes perfect sense

**Comment by choldgraf** [comment]
@astrojuanlu do you know if it's possible to do a hard-redirect on readthedocs? So that thebelab just goes straight to thebe's docs?

I tried to set up a redirect like this but not sure if it worked:

![image](https://user-images.githubusercontent.com/1839645/141178327-2e7fa436-8a55-4e1b-be72-cc15cf31af4c.png)

**Comment by pradyunsg** [comment]
Try having an exact registration from /en/latest/

That might work better, but it'll still only cover the documentation root.

**Comment by astrojuanlu** [comment]
The process to redirect an old Read the Docs site to somewhere else is a bit involved, but I tried my best to summarize it here: https://stackoverflow.com/a/69928404/554319

**Comment by choldgraf** [comment]
oof, hmm that is a lot...I think it might be better to just delete the old RTD site

**Comment by stevejpurves** [comment]
yes maybe just remove the old one! everything here points to the new docs and we'll soon be pushing through on the rest of the renaming in https://github.com/executablebooks/thebe/pull/508

(I'm not clear on when the move to the new URL happened though (did I manage to do this inadvertently in a recent change?) some of the content on there is recent 🤔 , especially as we've not brought in the PR that completes the renaming)

**Comment by stevejpurves** [comment]
@choldgraf are the `thebelab` docs under a different RTD project than the `thebe` ones?

**Comment by choldgraf** [comment]
OK so I did a quick GitHub search for `thebelab.readthedocs.io` and this actually turned up a decent amount of hits, so I think it'd be better to do a redirection so those links don't all break.

So I decided to follow [advice from @azonca's blog](https://zonca.dev/2020/09/redirect-readthedocs.html) and set up redirects with the following actions:

- I added an **Exact redirect** to point `thebelab.readthedocs.io/en/latest` to `thebe.readthedocs.io/en/latest`. This redirect will *only trigger if the server returns a 404 error*. So we need to *remove all pages from the `thebelab` docs, and make the `index` page do a hard-redirect.
- I've set up a temporary sphinx site at https://github.com/choldgraf/thebelab-temp and built the docs on readthedocs for thebelab. This will generate an empty site with one landing page that redirects to `thebe` docs.

So for example:

- thebelab.readthedocs.io should now redirect to thebe.readthedocs.io
- a direct page like https://thebelab.readthedocs.io/en/latest/ui.html should now redirect to https://thebe.readthedocs.io/en/latest/ui.html

It's not perfect, but I think it is good enough IMO, I'll close this issue!
